### PR TITLE
Add details for pluralization on extra fields message

### LINKED ISSUE #13411: [Form] Allowing plural message on extra data validation failure
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#34984
| PR author(s) | @popnikos
| Merged in    | 5.1-dev

### PR DESCRIPTION
**Pull request by jpjoao**
* Add ref anchor for Pluralization;
* Add link for Pluralization on extra fields message;

Fixes symfony/symfony-docs#13411

### CODE REVIEW

File: reference/forms/types/options/extra_fields_message.rst.inc [hunk 1]
```diff
@@ -7,3 +7,5 @@ This is the validation error message that's used if the submitted form data
 contains one or more fields that are not part of the form definition. The
```
Comment (HeahDude) [inline reference/forms/types/options/extra_fields_message.rst.inc:7]: Here, something like:
```rst
.. versionadded: 5.1

    The support for pluralization has been added in Symfony 5.1.
```

See https://symfony.com/doc/current/contributing/documentation/format.html#new-features-behavior-changes-or-deprecations, to learn more about our standards.

**Review by HeahDude — COMMENTED** [review]
It looks good, we just miss a `versionadded ` directive. Thank you!

File: reference/forms/types/options/extra_fields_message.rst.inc (hunk 1 above)
Comment (HeahDude) [reply reference/forms/types/options/extra_fields_message.rst.inc:7]: Ah I cannot comment on line 3.

Comment (jpjoao) [reply reference/forms/types/options/extra_fields_message.rst.inc:7]: Thanks for your feedback @HeahDude I will re-read the guidelines to make your life easier next time :p

I just added the required information.

File: reference/forms/types/options/extra_fields_message.rst.inc [hunk 2]
```diff
@@ -7,3 +7,9 @@ This is the validation error message that's used if the submitted form data
...
 placeholder ``{{ extra_fields }}`` can be used to display a comma separated
 list of the submitted extra field names.
+
+This message can be pluralized, see :ref:`formatting pluralized messages <component-translation-pluralization>` for details.
+
+    .. versionadded:: 5.1
```
Comment (HeahDude) [inline reference/forms/types/options/extra_fields_message.rst.inc:13]: This should be moved line 4 just after the title and a blank line and before the type.

File: reference/forms/types/options/extra_fields_message.rst.inc (hunk 1 above)
Comment (HeahDude) [reply reference/forms/types/options/extra_fields_message.rst.inc:7]: >I will re-read the guidelines to make your life easier next time

No worries ;)

**Review by HeahDude — APPROVED** [review]
Thanks! (one minor comment)

File: reference/forms/types/options/extra_fields_message.rst.inc (hunk 2 above)
Comment (jpjoao) [reply reference/forms/types/options/extra_fields_message.rst.inc]: Done.

**Comment by HeahDude** [comment]
Thank you @jpjoao.

**Pull request merged**